pygfx/objects/_more.py

```diff
@@ -270,14 +269,36 @@ class Text(WorldObject):
         rot_scale_transform="4x4xf4",
     )
 
-    def update_matrix(self, *args, **kwargs):
-        super().update_matrix(*args, **kwargs)
+    def __init__(
+        self,
+        geometry=None,
+        material=None,
+        *,
+        visible=True,
+        render_order=0,
+        render_mask="auto"
+    ):
+        super().__init__(
+            geometry,
+            material,
+            visible=visible,
+            render_order=render_order,
+            render_mask=render_mask,
+        )
+
+        # calling super from callback is possible, but slow so we register it as a second callback instead
+        self.world.on_update(super()._update_uniform_buffers)
+
+    @callback
+    def _update_uniform_buffers(self, transform: AffineBase):
```

Comment:
The `AffineBase` type here is `AffineTransform` in all other occasions, I suggest to replace it here to be consistent

Reply:
Yes, this should be `AffineBase` everywhere since I moved the callback system onto `AffineBase`. Originally it was just part of `AffineTransform` because it needed a way to update the `uniform_buffer` which lives on the WorldObject. Then, I learned that some objects use the old `update_world_matrix` as a callback for other logic, so I extended it.

Ideally, though, I would like to completely remove the callback mechanism (less complexity). This would require some refactoring though, and this PR does enough things as-is :)